frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -44,33 +47,41 @@ const Details_ = ({flags, obj: pvc}) => {
   const storageClassName = _.get(pvc, 'spec.storageClassName');
   const volumeName = _.get(pvc, 'spec.volumeName');
   const requestedStorage = _.get(pvc, 'spec.resources.requests.storage');
-  const storage = _.get(pvc, 'status.capacity.storage');
+  const volumeMode = _.get(pvc, 'status.volumeMode');
   const accessModes = _.get(pvc, 'status.accessModes');
   return <div className="co-m-pane__body">
     <SectionHeading text="PersistentVolumeClaim Overview" />
     <div className="row">
-      <div className="col-sm-6">
-        <ResourceSummary resource={pvc}>
+      <div className="col-sm-4">
+        <ResourceSummary resource={pvc} showPodSelector={false} showNodeSelector={false}>
           <dt>Label Selector</dt>
           <dd><Selector selector={labelSelector} /></dd>
         </ResourceSummary>
       </div>
-      <div className="col-sm-6">
+      <div className="col-sm-4">
         <dl>
           <dt>Status</dt>
-          <dd><PVCStatus pvc={pvc} /></dd>
+          <dd> { volumeName ? <React.Fragment> <PVCStatus pvc={pvc} /> to <ResourceLink kind="PersistentVolume" name={volumeName} /> </React.Fragment> : <PVCStatus pvc={pvc} /> } </dd>
```

Comment:
You should be able to use the `inline` prop to `ResourceLink` here, and I believe it avoids the CSS change you've made above.

Reply:
I was testing the suggestion you gave. The code looked like so -
<dd> { volumeName ? <React.Fragment> <PVCStatus pvc={pvc} /> to <ResourceLink kind="PersistentVolume" inline={true} name={volumeName} /> </React.Fragment> : <PVCStatus pvc={pvc} /> } </dd>


This still did not resolve the issue as it looks like this now.
![Screenshot from 2019-03-21 15-24-23](https://user-images.githubusercontent.com/27074500/54744679-9f181a00-4bed-11e9-8fb8-dd0d2f3dc351.png)


Is it happening because of using it along with "PVCStatus"?